pkgs/development/python-modules/deploykit/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, bash
+, openssh
+, pytestCheckHook
+, stdenv
+, fetchpatch
+}:
+
+buildPythonPackage rec {
+  pname = "deploykit";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "numtide";
+    repo = pname;
+    rev = version;
+    hash = "sha256-mQgv373BIlyNbm2+F6BqqdJkkJn4Cy5FNwQNAyBb/h4=";
+  };
+
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/numtide/deploykit/commit/44e3f12f2e65d11208e8809127dfe9a171a96db0.patch";
+      hash = "sha256-5Q9oRKhvVe3IrErHj/OTP3hKNsZCdRR0EV4XjdqA/us=";
+    })
```

Comment:
What's this patch for and should it be removed again at some point?

A comment would be helpful.

Reply:
I just made another release 1.0.1... This means the patch can be dropped: https://github.com/numtide/deploykit/releases/tag/1.0.1